packages/hop-node/src/watchers/BondTransferRootWatcher.ts

```diff
@@ -108,6 +108,9 @@ class BondTransferRootWatcher extends BaseWatcher {
         `transferRootHash ${transferRootHash} already bonded. skipping.`
       )
       const event = await l1Bridge.getTransferRootBondedEvent(transferRootHash)
+      if (!event) {
```

Comment:
I was seeing this particular event be null randomly when syncing so destructuring would fail so wanted to make the error more friendly. The other places already have null event handling